src/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -86,6 +87,13 @@ public abstract partial class XmlReader : IDisposable
 
         internal const int AsyncBufferSize = 64 * 1024; //64KB
 
+        private static volatile XmlReaderSettings s_defaultReaderSettings;
+
+        private static XmlReaderSettings DefaultReaderSettings =>
```

Comment:
It would be better to make it a regular readonly static in XmlReaderSettings . It would make the manual lazy initialization dance unnecessary.

Reply:
The manual lazy initialization should not be needed even when it is here because of XmlReader has no other statics today, but XmlReader is a big class and so it is hard to see that.

Reply:
I'll move it to XmlReaderSettings. Thanks.